identity_core/src/common/ordered_set.rs

```diff
@@ -108,6 +108,11 @@ impl<T> OrderedSet<T> {
     self.0
   }
 
+  /// Returns a mutable reference to the underlying vector.
+  pub fn as_vec_mut(&mut self) -> &mut Vec<T> {
+    &mut self.0
+  }
```

Comment:
This is unnecessary and can be removed.

If kept it should be `#[inline]` and be named `as_vec_mut_unchecked`, similar to `iter_mut_unchecked`.

```suggestion
```